### PR TITLE
Add full website

### PR DESCRIPTION
**Pull request by ahmed-alfarouq**
What I did:
- Added navbar
- Added routing
- Added home page
- Added calculator page
- Added quote page
- Make the site resposive

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @ahmed-al-farouq 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import React, { useState } from 'react';
+import {
+  BrowserRouter,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
```
Comment (mikenath223) [inline src/App.js:7]: - [ ] Kindly deploy the project online and include the live deploy link in the readme.

File: .eslintrc.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
-    "no-shadow": "off"
+    "no-shadow": "off",
+    "react/display-name": "off"
```
Comment (mikenath223) [inline .eslintrc.json]: - [ ] You seem to have added an extra eslint rule here, please remove the `react/display-name: off` rule.

File: src/App.js (hunk 1 above)
Comment (ahmed-alfarouq) [reply src/App.js:7]: Hi @mikenath223,
It's not one of the requirements.
Thanks for the feedback

File: .eslintrc.json (hunk 2 above)
Comment (ahmed-alfarouq) [reply .eslintrc.json]: Hi @mikenath223,
That's right. I will do that.
Thanks for the feedback.

**Review by mke2111 — APPROVED** [review]
### Hi @ahmed-al-farouq, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**